card/lib/card/loader.rb

```diff
@@ -21,6 +21,40 @@ def load_mods
         load_set_patterns
         load_formats
         load_sets
+
+        if ENV['RAILS_ENV'] == "development"
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.